core/src/main/java/it/gov/pagopa/rtd/transaction_filter/service/store/AggregationData.java

```diff
@@ -0,0 +1,25 @@
+package it.gov.pagopa.rtd.transaction_filter.service.store;
+
+import lombok.Data;
+import lombok.ToString;
+
+@Data
+@ToString
+public class AggregationData {
+
+  private short numTrx;
+  private int totalAmount;
```

Comment:
Please add a comment, the 'int' here is safe until the aggregations are made on daily basis.